docs/workflow/testing/libraries/testing.md

```diff
@@ -1,5 +1,8 @@
 # Testing Libraries
 
+**WARNING** Instructions on this page appear to be unmaintained and do not work as written. See [here](https://github.com/dotnet/runtime/issues/54323) for details.
```

Comment:
If this is true we should just delete it :)

Reply:
I'm fine with either of those options but I didn't want to presume. This felt like the minimal change that would warn others not to rely on it while we decided the fate of this page.